ci/tasks/scripts/with-docker.sh

```diff
@@ -9,7 +11,7 @@ function load_layers() {
 
   # don't do anything if we don't have cache
   [[ ! -d 'docker-cache' ]] && return 0;
-  [[ -z "$(ls docker-cache)" ]] && return 0;
+  [[ -z "$(ls docker-cache/*/image.tar)" ]] && return 0;
 
   echo "loading docker layer cache"
   pids=
```

Comment:
As an aside, `wait` can take multiple process IDs, so you can do `wait "${pids[@]}"` below instead of the `for` loop

Reply:
that's good to know, will apply this in a separate PR